src/coreclr/tools/aot/crossgen2/crossgen2.csproj

```diff
@@ -59,4 +57,19 @@
       </ItemGroup>
   </Target>
 
+  <Target Name="LocateNativeCompiler"
+          Condition="'$(RunningPublish)' == 'true' and '$(HostOS)' != 'windows'"
+          BeforeTargets="SetupOSSpecificProps">
+      <PropertyGroup>
+        <CppCompilerAndLinker Condition="'$(CppCompilerAndLinker)' == ''">clang</CppCompilerAndLinker>
+      </PropertyGroup>
+
+      <Exec Command="sh -c 'build_arch=&quot;$(TargetArchitecture)&quot; compiler=&quot;$(CppCompilerAndLinker)&quot; . &quot;$(RepositoryEngineeringDir)/common/native/init-compiler.sh&quot; &amp;&amp; echo $CC' 2>/dev/null"
```

Comment:
looks like this is resolving gcc now in the `runtime (Build Installer Build and Test coreclr Linux_arm64 Release` build:

```
  Generating native code
gcc : error : unrecognized command line option ‘--target=aarch64-linux-gnu’ [/__w/1/s/src/coreclr/tools/aot/crossgen2/crossgen2.csproj]

```

Reply:
This is only failing after merge from `main` because property name was changed there: https://github.com/dotnet/runtime/commit/8e15d9f523209caf08db57897777b7beb0516c99